.github/workflows/comparison-results.yml

```diff
@@ -0,0 +1,145 @@
+name: Monitoring Stability and Comparing Results
+
+# Triggers when a pull_request or a push action is configured on master branch
+on:
+  push:
+    branches: ["main", "dev", "comparison-result-generation"]
+  pull_request:
+    branches: ["main", "dev", "comparison-result-generation"]
+
+jobs:
+  generate_comparison_results:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Folder to store reports
+        run: mkdir ${GITHUB_WORKSPACE}/reports
+
+      - name: Download CPU and Memory usage capture tool
+        run: mkdir ${GITHUB_WORKSPACE}/tools && curl https://raw.githubusercontent.com/Privado-Inc/privado-core-results-monitoring/main/capture_usage.sh --output ${GITHUB_WORKSPACE}/tools/capture_usage.sh
+
+      - name: Start monitoring cpu and memory usage
+        run: bash ${GITHUB_WORKSPACE}/tools/capture_usage.sh &
+
+      # Setup python 3.8 in the pipeline
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+
+      # Setup privado environment in the pipeline
+      - name: Setup Privado Env
+        run: curl -o- https://raw.githubusercontent.com/Privado-Inc/privado-cli/main/install.sh | bash
+
+      - name: Download the comparison tool
+        run: curl https://raw.githubusercontent.com/Privado-Inc/privado-core-results-monitoring/main/compare.py --output ${GITHUB_WORKSPACE}/tools/compare.py
+
+
+      # For every repository, clone, scan and store the generated JSON
+      # For Privado-Demo/privado-analytics
+      - name: Cloning Privado-Demo/privado-analytics
+        uses: actions/checkout@v2
+        with:
+          owner: "Privado-Demo"
+          repository: "Privado-Demo/privado-analytics"
+          path: ./repos/privado-analytics
+
+      # cannot simulate restarting the terminal behavior in the pipeline, so navigated to the installation path to run the tool
+      - name: Scan karan-batavia/privado-analytics (stable) using privado scan
+        run: ~/.privado/bin/privado scan --skip-upload ${GITHUB_WORKSPACE}/repos/privado-analytics
+
+      - name: Store privado-analytics.json (stable)
+        run: mkdir ${GITHUB_WORKSPACE}/stable_results && cat ${GITHUB_WORKSPACE}/repos/privado-analytics/.privado/privado.json > ${GITHUB_WORKSPACE}/stable_results/privado-analytics.json
+
+      - name: Scan karan-batavia/privado-analytics (dev) using privado scan
```

Comment:
Same here. Replace `karan-batavia`

Reply:
Sure, will ask Prashant or Pandurang to add the organization name to Github secrets.

Reply:
@karan-batavia We do not need to add that in secret. Please refer [default envs](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) for GH actions. I'm guessing you will be using `GITHUB_REPOSITORY_OWNER`.

Reply:
```GITHUB_REPOSITORY_OWNER``` gives the current repository owner name, these repos are on different repositories